app/views/reservations/my_reservations.html.erb

```diff
@@ -1,11 +1,12 @@
-<!-- app/views/reservations/my_reservations.html.erb -->
-<h1>My Reservations</h1>
-<ul>
- <% @reservations.each do |reservation| %>
-   <li>
-     <h2><%= reservation.item_name %></h2>
-     <p>Date: <%= reservation.date %></p>
-     <p>City: <%= reservation.city %></p>
-   </li>
- <% end %>
-</ul>
+<div class="container">
+  <h1 class="my-4">My Reservations</h1>
+  <ul class="list-group">
+    <% @reservations.each do |reservation| %>
+      <li class="list-group-item">
+        <h2 class="h5"><%= reservation.item_name %></h2>
+        <p>Date: <%= reservation.date %></p>
+        <p>City: <%= reservation.city %></p>
+      </li>
+    <% end %>
+  </ul>
+</div>
```

Comment:
- [ ]The views are rendering though react not ruby on rails
- [ ]There are no routes to this view in the routes file